### DISCUSSION TITLE
why propose Google search engine?

### DISCUSSION THREAD

**Discussion by ayoahha** [bromite/bromite#2288] [Ideas]
I'm wondering, given that bromite is privacy focused, why it comes with the "Google search engine" by default ? (Or any other Gafam related search engines...)

Why not propose only privacy focused search engines by default like DDG, SearX, metager... like in fact ungoogled chromium?

**Comment by ghost** [comment]
Have you ever read the [FAQ](https://github.com/bromite/bromite/blob/master/FAQ.md)

**Reply by ayoahha** [reply]
Yes, and ?
